ontopy/ontology.py

```diff
@@ -100,8 +100,9 @@ def get_ontology(
                 this Ontology subclass.  Defaults to `ontopy.Ontology`.
             label_annotations: Sequence of label IRIs used for accessing
                 entities in the ontology given that they are in the ontology.
-                Label IRIs not in the ontology will simply be ignored.
-                Detault to DEFAULT_LABEL_ANNOTATIONS module variable.
+                Label IRIs not in the ontology will need to be added to
+                ontologies in order to be accessible.
+                Default is None.
```

Comment:
None (default), is equivalent to set `label_annotations` to the `DEFAULT_LABEL_ANNOTATIONS` module variable.